distribution/io/src/assemble/io.xml

```diff
@@ -47,7 +47,7 @@
 
     <file><source>${basedir}/../../pulsar-io/cassandra/target/pulsar-io-cassandra-${project.version}.nar</source></file>
     <file><source>${basedir}/../../pulsar-io/twitter/target/pulsar-io-twitter-${project.version}.nar</source></file>
-    <file><source>${basedir}/../../pulsar-io/kafka/target/pulsar-io-kafka-${project.version}.nar</source></file>
```

Comment:
@sijie I might be missing some context, what it can break? The nar file name/source path is changed.
Similar change is https://github.com/apache/pulsar/pull/9808 - am I missing some context that will help me understand why that one was safe and this one is not?
I relied on integration tests to catch potential breaks, please let me know if there is something the tests won't catch, how can I confirm that everything works fine in this case?

Reply:
I missed #9808. There is no breaking change here.

#9808 is fine because the adopter is used for both standalone connector and a dependency for debezium connector. This goes to my major comment. I don't think you should put the Kafka sink connector wrapper in Kafka connector. It should go to the `pulsar-io-kafka-connect-adaptor` module.